tests/components/test_logbook.py

```diff
@@ -7,48 +7,63 @@
 # pylint: disable=protected-access,too-many-public-methods
 import unittest
 from datetime import timedelta
-
 import homeassistant.core as ha
+
 from homeassistant.const import (
     EVENT_STATE_CHANGED, EVENT_HOMEASSISTANT_START, EVENT_HOMEASSISTANT_STOP)
 import homeassistant.util.dt as dt_util
 from homeassistant.components import logbook
 
-from tests.common import get_test_home_assistant, mock_http_component
+from tests.common import mock_http_component
 
 
 class TestComponentHistory(unittest.TestCase):
     """ Tests homeassistant.components.history module. """
 
-    def test_setup(self):
+    def setUp(self):
         """ Test setup method. """
-        try:
-            hass = get_test_home_assistant()
-            mock_http_component(hass)
-            self.assertTrue(logbook.setup(hass, {}))
-        finally:
-            hass.stop()
+        self.hass = ha.HomeAssistant()
+        mock_http_component(self.hass)
+        logbook.setup(self.hass, {})
+
+    def tearDown(self):
+        self.hass.stop()
+
+    def test_service_call_create_logbook_entry(self):
+        self.assertTrue(self.hass.services.call('logbook', 'log', {
+            logbook.ATTR_NAME: 'test',
+            logbook.ATTR_MESSAGE: 'test',
+            logbook.ATTR_DOMAIN: 'test',
+            logbook.ATTR_ENTITY_ID: 'test',
+        }, blocking=True))
+
+    def test_service_call_create_logbook_entry_missing_parameter(self):
+        self.assertTrue(self.hass.services.call('logbook', 'log', {
+            logbook.ATTR_MESSAGE: 'test',
+        }, blocking=True))
```

Comment:
You're not testing if the event that is created matches your parameters. Add an event listener for logbook_entry that captures calls in a list. Wait till all jobs are finished with hass.pool.block_till_done() and test the call